### DISCUSSION TITLE
是否可以考虑直接拉取网页来作为知识库

### DISCUSSION THREAD

**Discussion by shoufei403** [lobehub/lobehub#4097] [Ideas]
是否可以考虑直接拉取网页来作为知识库，就像cursor的docs功能那样。

直接给python documentation的网页链接，cursor就可以把整个文档拉取下来，并且直接indexing为知识库。

我觉得这个功能很酷。是否考虑在lobechat中实现该功能？

**Comment by arvinxx** [comment]
有考虑的，后续会做
